connect-another-device/connect-another-device.js

```diff
@@ -4,15 +4,25 @@ module.exports = {
   name: 'Should the user see the "connect another device" screen',
   hypothesis: 'A nudge to connect another device will help increase multi-device users',
   startDate: '2015-01-01',
-  subjectAttributes: ['uniqueUserId', 'isMetricsEnabledValue', 'forceExperimentGroup'],
+  subjectAttributes: ['uniqueUserId', 'forceExperiment', 'forceExperimentGroup'],
   independentVariables: ['connectAnotherDevice'],
```

Comment:
@shane-tomlinson could you also add `false` for this value into https://github.com/mozilla/fxa-content-experiments/blob/dev/defaults.json